.circleci/config.yml

```diff
@@ -364,6 +364,38 @@ jobs:
           path: /go/src/github.com/Azure/acs-engine/_logs
       - store_artifacts:
           path: /go/src/github.com/Azure/acs-engine/_output
+  openshift-3.9-release-e2e:
+    working_directory: /go/src/github.com/Azure/acs-engine
+    docker:
+      - image: quay.io/deis/go-dev:v1.9.1
+        environment:
+          GOPATH: /go
+    steps:
+      - checkout
+      - run: |
+          echo 'export TIMEOUT=20m' >> $BASH_ENV
+          echo 'export SKIP_TEST=true' >> $BASH_ENV
+          echo 'export LOCATION=eastus' >> $BASH_ENV
+          echo 'export ORCHESTRATOR_RELEASE=3.9' >> $BASH_ENV
+          echo 'export CLUSTER_DEFINITION=examples/openshift.json' >> $BASH_ENV
+          echo 'export CREATE_VNET=false' >> $BASH_ENV
+          echo 'export CLEANUP_ON_EXIT=false' >> $BASH_ENV
+          echo 'export RETAIN_SSH=false' >> $BASH_ENV
+          echo 'export SUBSCRIPTION_ID=${SUBSCRIPTION_ID_E2E_OPENSHIFT}' >> $BASH_ENV
```

Comment:
@jackfrancis @jim-minter @pweil- I am using our own envs to test but we will need to sync on who is going to control these IDs (I guess MS). In that case we will likely need to start using our published node images so they can be accessed by the CI subscription, does that sound right?

Reply:
yes, we should ensure we can white list the CI subscription for our images
/cc @kennywoodson

Reply:
If you send me the subscription ids to be shared I can add them in the portal.